### ISSUE TITLE
Let learning rate be a hyperparameter

### ISSUE THREAD

**Issue by abhay-venkatesh** [abhay-venkatesh/segnet#13]
No need to feed it into the network

**Comment by abhay-venkatesh** [comment]
Closing because of priorities.
